docs/manual/java/guide/services/AdditionalRouters.md

```diff
@@ -2,7 +2,7 @@
 
 Since Lagom 1.5.0, it is possible to extend a Lagom Service with additional Play Routers.
 
-This is particularly useful when integrating Lagom with existing Play Routers, for instance a [Play gRPC Router](https://developer.lightbend.com/docs/play-grpc/0.5.0/lagom/serving-grpc.html) , or any other Play router that you have at your disposal.
+This is particularly useful when integrating Lagom with existing Play Routers, for instance a [Play gRPC Router](https://developer.lightbend.com/docs/play-grpc/0.5.0/serving-grpc.html?language=java), or any other Play router that you have at your disposal.
```

Comment:
This link is wrong.

I introduced the broken fix with the `/lagom` segment without noticing the `0.5.0` segment.

The correct link is:

https://developer.lightbend.com/docs/play-grpc/0.6.0/lagom/serving-grpc.html?language=java

Note `0.6.0` and `lagom` segments on the path.

Reply:
```suggestion
This is particularly useful when integrating Lagom with existing Play Routers, for instance a [Play gRPC Router](https://developer.lightbend.com/docs/play-grpc/0.6.0/lagom/serving-grpc.html?language=java), or any other Play router that you have at your disposal.
```